benches/benches/block_target_gas.rs

```diff
@@ -29,6 +38,27 @@ use fuel_core_types::{
     fuel_types::AssetId,
 };
 
+/// Allocates a byte array from heap and initializes it. Then points `reg` to it.
+fn aloc_bytearray<const S: usize>(reg: u8, v: [u8; S]) -> Vec<Instruction> {
```

Comment:
It would be nice to move these functions to `utils` and reuse them instead of duplicating them.